docs/docs/1.3.0/policies/mutual-tls.md

```diff
@@ -189,8 +189,57 @@ A few considerations:
 
 ### Intermediate CA
 
-You can also work with an Intermediate CA with a `provided` backend. As with the root CA, upload the certificate and key as secrets, then reference 
-the Secrets in the mTLS configuration. In the `cert` section, specify the certificate from an Intermediate CA first, and then the certificate from the Root CA. Separate the items by an empty line.
+You can also work with an Intermediate CA with a `provided` backend. Generate the certificate and place it before the certificate from the root CA, in the same file with an empty line separating the items. Then create the secret to specify in the `cert` section of the config. The secret for the `key` should contain only the private key of the certificate from the intermediate CA.
+
+You can chain certificates from multiple intermediate CAs the same way. Place the certificate from the closest CA at the top of the cert file, followed by certificates in order up the certificate chain, then generate the secret to hold the contents of the file.
+
+Sample certificate file for a single intermediate CA:
+
+```
+-----BEGIN CERTIFICATE FROM INTERMEDIATE CA-----
+MIIDdjCCAl6gAwIBAgICEAEwDQYJKoZIhvcNAQELBQAwRDELMAkGA1UEBhMCR0Ix
+EDAOBgNVBAgMB0VuZ2xhbmQxEjAQBgNVBAoMCUFsaWNlIEx0ZDEPMA0GA1UEAwwG
+S3VtYUNBMB4XDTIxMDUxMjEzMzU1MVoXDTMxMDUxMDEzMzU1MVowUDELMAkGA1UE
+BhMCR0IxEDAOBgNVBAgMB0VuZ2xhbmQxEjAQBgNVBAoMCUFsaWNlIEx0ZDEbMBkG
+A1UEAwwSS3VtYUludGVybWVkaWF0ZUNBMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8A
+MIIBCgKCAQEA1VzY9vOr8+SINzqA8Rwk4bpeex32Zn9BGAUTweRgomQC7Yfzrm6/
+Vk74/T/46n3FydpdEZTdoFKCF8EsA0eqAEfWi6tu7D41GOUFUYpdRJBJEq+HE17Q
+N8SFMquy8NhCtK8th8ytSu2ThvCOq1MHT5WjtQUmRGSJMlcfWA5TsCIK0Sb3cSf3
+jadjEqcmcvJN6Xa0Y0VivcPg5eB+We7BNnp4ogqmZw0veoPjc14HVZpqxrra9Yez
+DRai6rnHqDjnkMMhe9MmSkCKD9Ldwduq0ZfuOQFIBOaX+4MKUyDN4tTMCcRRl/Nl
+A4JgrNNWCFfUQV0VmQ0Tc8+cn/+gokHAZwIDAQABo2YwZDAdBgNVHQ4EFgQUGNjz
+Te727HX4AqZDMn1L9XzkTaYwHwYDVR0jBBgwFoAUSu2E4Ue5aPzdWQCCNp36Pf3i
+YbcwEgYDVR0TAQH/BAgwBgEB/wIBADAOBgNVHQ8BAf8EBAMCAQYwDQYJKoZIhvcN
+AQELBQADggEBACuOczJlf4wcT9rfAIrZHuI5aCzYTKOxJllhN5e/eEhMYpsox6Zb
+4CZXS3wdJ3fVugddLWDzIAjrNE1DrOpugUPurNIpHsT6u+SHFXkRsXyHFfMA+CZJ
+0tOYEtP1r3BnqsY/nh0GJqHJxaJolEaqFaKgKTQPTinOxTKFxsHa1OHlsvkdxvot
+d2BQhPQYWes3LMPxtGhS5kwKaXaB3gzTnzjGvgGNeJ+l0AiWqXkivixpox3/6mMa
+90mwssl4sRQQLR1kLFU4hwghNm52Pk7o7HSTEXsnB+ZhHB9skpetY6R4uKWh8xap
+Xmj4PDrAA5OKZzSO7Yhdt0vXPOIrjShMxvA=
+-----END CERTIFICATE-----
+
```

Comment:
This line is not needed. I see how the description can be confusing.

When I say that they should be separated by new line I mean this
```
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE FROM ROOT CA-----
```
If they were not separated, it would be this
```
-----END CERTIFICATE----------BEGIN CERTIFICATE FROM ROOT CA-----
```

btw. I'm not sure it's legal to put `FROM INTERMEDIATE CA` and `FROM ROOT CA`.